odata-protocol/11.4 Data Modification.md

```diff
@@ -567,21 +569,29 @@ Content-Type: application/json
 ```
 
 If the targeted entity in the payload contains some structural properties,
-`PUT` resets all its other structural properties. The following therefore does
-not have the desired effect:
+`PUT` resets all its other structural properties. The following alternative
+payloads both reset the first and last names of the new manager:
 ```json
-PUT http://host/service/Employees(7)
-Content-Type: application/json
-
 {
   "FirstName": "Suzanne",
   "LastName": "Brown-Johnson",
   "Manager": {
+    "@context": "$metadata#$ref",
     "@id": "Employees(6)",
     "EmployeeID": 6
```

Comment:
That is not a valid entity reference, additional properties are not allowed.

Reply:
If you remove the `@context`, it becomes an incomplete entity and a valid example for undesired effects.